README.md

```diff
@@ -0,0 +1,34 @@
+# Ebay Clone Rails App
+Ebay clone that offers a marketplace for buying and selling items.
+
+## Requirement
+- ruby ~> 2.2.1
+- bundler >= 1.3.0, < 2.0
+- rake >= 0.8.7
```

Comment:
rake is in your Gemfile and will be installed when you `bundle install`, so this is not necessary to call out here.